### DISCUSSION TITLE
Is it possible to link a word of a page to a word of another page ?

### DISCUSSION THREAD

**Discussion by AlbyTree** [zim-desktop-wiki/zim-desktop-wiki#2486] [Q&A] [answered]
In this [article](https://zim-wiki.org/manual/Help/Links.html#link-to-a-heading-or-object), it seems that whatever has an ID can be referenced but I don't know how to explicitly assign an ID to a word.

**Comment by jaap-karssenberg** [comment]
As mentioned in the manual page you refer: there is a special "anchor"
object to place a link anywhere in the text or to e.g. set an explicit id
for a heading. This can be inserted by typing "##name"

So you can not refer an individual word, but you need to create an anchor
to which you can link

**Comment by AlbyTree** [comment]
Mh, I still don't understand how it should work.
For simplicity, in the same page I have
```
LandingLink

StartLink
```
and I want to click on ```StartLink``` to get to ```LandingLink```: I add a link to ```StartLink``` as ```##LandingLink``` but when I click ```StartLink``` I get the error ```Id landinglink not found on this page``` .
How can I create an anchor ?

**Comment by tridian-tn** [comment, marked as answer]
@AlbyTree I've had a go at this, and I get the same error as you if the anchor has capitals in it. If you then copy/paste the link, or type the link in by hand, the link is always tried as a lowercase name. GIF attached because I'm not explaining myself well! It looks like a bug: anchors should be consistently treated by Zim as either case-sensitive, or case-insensitive. 

![Zim](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/10810780/807fb50b-2ec4-4f20-a062-e66046677bc3)

**Comment by AlbyTree** [comment]
Ah, so that's how you create anchors! I think the wording of how to create an anchor in the wiki should be more clear, because I thought that I had to put the ```##``` in the insert link menu, not in the actual page.
(Moreover, there was no glyph rendered in Edge)
![zim-anchor](https://github.com/zim-desktop-wiki/zim-desktop-wiki/assets/34436621/ec3d01b7-ae80-4474-9af2-2f00fc65bc2b)
The GIF of @tridian-tn answer my question... I don't know if the problem of case-sensitive anchors should be put in another post.

**Reply by introt** [reply]
Feel free to add it to the #1942 pile :)

**Comment by Lost-Projects** [comment]
I just logged in to say thanks for helping me to solve the anchors problem. I've been trying to make them work for a while. But i had to do a bit of working it out for myself and I thought I'd add a summary. Feel free to check and correct. I'm using version 0.74.3.

1. An anchor is made by typing ##name-of-anchor (I think lower case and dashes for spaces is correct) This is rendered in the source (or type this directly) as {{id: name-of-anchor}} and creates the symbol (don't know the name) in the gif above. You can add text or a heading after it if required.

2. From within the same page, you can link to this anchor using, [[#name-ofanchor|Click here to jump to the anchor]]

3. From another page, the link format is, [[example_page_with_anchor#name-ofanchor|Click here to jump to an anchor on another page]]

Note that in the link editing dialog box, use of the # results in the field turning red as if the syntax is wrong, but it works for me and I ignore this warning.

I got anchors working for me with some source editing and quick tests, so there may be errors in the summary. If typing links directly, it my be necessary to reload the page to activate them. Again, feel free to correct.

---
As an attempt to answer the original question, it might be possible to use @tagging to label all pages with a particular word and find them with a search, but if that's not the goal, then linking a section of text (e.g. a word) to a section of text in another page (could be the same word) is possible by placing an anchor by the specific text. Mutual links would also be possible so two examples of a word could link to each other or create a "chain" of linked words across a number of pages, but of course I can't see how to link a word to multiple examples - that's more of a search function.
